src/lib/Form/Type/Extension/ModifyFieldDefinitionsCollectionTypeExtension.php

```diff
@@ -22,22 +23,38 @@ final class ModifyFieldDefinitionsCollectionTypeExtension extends AbstractTypeEx
 {
     private string $fieldTypeIdentifier;
 
+    /** @var string[] */
+    private array $fieldIdentifiers;
+
     /** @var array<string, mixed> */
     private array $modifiedOptions;
 
+    private ?SpecificationInterface $contentTypeSpecification;
+
     /**
-     * @param string $fieldTypeIdentifier
+     * @param string|string[] $fieldIdentifiers
      * @param array<string, mixed> $modifiedOptions
      */
-    public function __construct(string $fieldTypeIdentifier, array $modifiedOptions)
-    {
+    public function __construct(
+        string $fieldTypeIdentifier,
+        array $modifiedOptions,
+        $fieldIdentifiers = [],
```

Comment:
Why are there two parameters that are named in the exact same way (`$fieldIdentifier` and `$fieldIdentifiers`)?

Reply:
what about deprecating passing `string` into it, and just keep array for 5.0?
@ibexa/php-dev